Expecto/Expecto.fsproj

```diff
@@ -151,49 +151,6 @@
     </When>
   </Choose>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v2.0' Or $(TargetFrameworkVersion) == 'v3.0')">

```

Comment:
These were removed by `paket` automatically because I [upgraded and restricted the target frameworks for `Mono.Cecil`](https://github.com/haf/expecto/pull/101/files#diff-b27553841b4413a24ed1b9aff865105bR16):

```diff
  nuget BenchmarkDotNet.Toolchains.Roslyn = 0.9.9
  nuget BenchmarkDotNet.Core = 0.9.9
  nuget NuGet.CommandLine
 -nuget Mono.Cecil
 +nuget Mono.Cecil = 0.10.0-beta2 framework: net40
  
  group build
      source https://www.nuget.org/api/v2
```